kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -320,6 +340,36 @@ public String toString() {
     }
   }
 
+  private static class CannedAclValidator implements ConfigDef.Validator {
+    public static final Map<String, CannedAccessControlList> ACLS_BY_HEADER_VALUE = new HashMap<>();
+    public static final String ALLOWED_VALUES;
```

Comment:
These are both now `final` and renamed to all caps. I also made ALLOWED_VALUES public.